config/promxy.yaml

```diff
@@ -1,37 +0,0 @@
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: management-cluster-promxy-config
-  namespace: giantswarm
-data:
-  values.yaml: |
-    promxy: 
```

Comment:
Can this be moved to the installation repo?

Reply:
It will be as part of this transition: https://github.com/giantswarm/config/pull/155/files#diff-09c5104a5f568fe2504639817b6647163329b60d0634345b2b328fcda329f66aR6

Reply:
@kubasobon I think that's enough to move that to values.yaml here.

Reply:
Correct me if I'm wrong: I understood that this ConfigMap is used exclusively in MCs. That's why it should be configured in `giantswarm/config`. If `promxy-app` is installed in WC, this part of config should not appear. 
That's why it's not in `values.yaml`.

Reply:
Yes this is exactly it :)

Reply:
Thanks @kubasobon